backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxGenerator.java

```diff
@@ -177,6 +182,8 @@ private void fillReportDocument(
             fillOverview3rdPartyComponentTable(document, projectLicenseInfoResults);
             fillAdditionalRequirementsTable(document, obligationResults);
 
+            fillCommonRulesTable(document, project);
```

Comment:
Looking at the table indices above, I would suggest, that the common rules table should be before the additional requirements table in the final document. Again, it would be easier for other developers if the method order could reflect the order in the document...

Reply:
Yeah, right. I will fix this